recipes/ptflops/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ptflops" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/95/3d/d2af80d6b90d74f1b73e688c287434912fc4416517c6f7505bb6d2ca77d1/ptflops-0.7.3.tar.gz
+  sha256: 59c0af2505255108df019674fb805eb61178f0baad6515720bb30dcb29592071
+
+build:
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - torchvision
```

Comment:
`torchvision` doesn't looks like it's required to build the package, so it should be removed from the `host` section